precompiles/xcm-utils/src/tests.rs

```diff
@@ -221,7 +221,7 @@ fn test_send_clear_origin() {
 		precompiles()
 			.prepare_test(CryptoAlith, Precompile1, input)
 			// Fixed: TestWeightInfo + (BaseXcmWeight * MessageLen)
-			.expect_cost(100001000)
+			.expect_cost(100000000)
 			.expect_no_logs()
```

Comment:
TODO: check this change. Seems like `ClearOrigin` is not being accounted in costs.

Reply:
Checked, it's because xcm execution weight is not charged any more in `xcm_send` extrinsic.

Associated PR: https://github.com/paritytech/polkadot/pull/7201